### ISSUE TITLE
Versionpress + Polylang = :(

### ISSUE THREAD

**Issue by pryley** [versionpress/support#7]
Polylang allows you to change the directory name in pretty permalinks to set the language.

For example, the setting will modify the site URL from "http://example.com/" to "http://example.com/en/"

Unfortunately, the Versionpress REST URL doesn't take this into account so the admin interface does not load correctly.

**Comment by borekb** [comment]
Thanks for the report, we will look into it. It's true that we didn't test for this scenario.

**Comment by pavelevap** [comment]
@pryley: Hi, I tested latest VersionPress 2.2, WordPress 4.4 and Polylang 1.7.12. I activated Polylang and switched to directory structure, then published one post and everything works as expected, changes are correctly tracked and displayed in administration. Do you have latest version of VersionPress? There were some problems with rewrite rules in older versions.

**Comment by pryley** [comment]
@pavelevap I downloaded v2.2 and tested again and it's now working fine.

I was curious to see what changed so I ran a diff, but I couldn't see anything that would have affected the rewrites. I switched back to v2.1.1 and...it was also working. 

I'm not sure what changed. I remember that after activating versionpress, the Versionpress page was stuck on the loading animation and the console was full of asset loading errors with URLs beginning with, "http://example.dev/vp-json/...". I only had a few plugins active, and the first thing I thought was that versionpress was trying to load assets using its API (as it wasn't loading directly from an admin URL). I knew that Polylang was rewriting the site URL so I first tried refreshing the permalink settings. That didn't work, so the next thing I did was deactivate Polylang which made the Versionpress page load correctly.

After that I reactivated Polylang and deactivated Versionpress. Perhaps deactivating/reactivating Polylang fixed it (I didn't check Versionpress again after reactivating Polylang). Maybe it was a local cache issue. 

Anyway, I'm closing this issue. Thanks for looking into it.
